### DISCUSSION TITLE
How to make this key

### DISCUSSION THREAD

**Discussion by LittleButterflyIsFly** [solokeys/solo2#77] [Q&A]
Hello 

I am a beginner in the field of making things such as this but I want to learn how to. The guide is a bit confusing for me. How do I exactly use this LPC55S69-EVK. I don't quite understand on how you use this as a USB to plug into something or do I connect my USB to this? 

I am sorry if this post is not supposed to go here

Thank you very much for any feedback!

**Comment by conorpp** [comment]
The dev board, LPC55S69-EVK, contains the LPC55S69 microcontroller and a debugger on it.  You can connect over USB to the debugger to program the LPC55S69 chip.  There's also another USB port that is the "normal" USB connector for the LPC55S69 chip, which would be for communicating with the application.

**Reply by LittleButterflyIsFly** [reply]
So basically I program the chip which is on the board and then I use the board as my fido key? or am I able to put it on a usb?

Or does the usb have to have a chip in order to function as a FIDO key

**Reply by conorpp** [reply]
Yes the dev board is all you need to develop with.  You could use it as a normal fido key but it's an awkward form factor.  Best for development.

**Reply by LittleButterflyIsFly** [reply]
How could I make something with a more usable form factor?

**Reply by conorpp** [reply]
You could learn some design tools to make a design that you can submit for fabrication, or hire someone else to do it.

**Reply by LittleButterflyIsFly** [reply]
What design tools do you recommend?

**Reply by conorpp** [reply]
Kicad :)
